spec/shared/json_convertible_spec.rb

```diff
@@ -25,9 +25,26 @@ def initialize(mock_json_attribute: nil)
   end
 end
 
+class MockArrayJSONConvertible
+  include FastlaneCI::JSONConvertible
+
+  # @return [Array(MockJSONConvertible)]
+  attr_accessor :mock_json_array_attribute
+
+  def initialize(mock_json_array_attribute: nil)
+    self.mock_json_array_attribute = mock_json_array_attribute
+  end
+end
+
 module FastlaneCI
   describe JSONConvertible do
     let (:mock_object) { MockJSONConvertible.new(one_attribute: "Hello", other_attribute: Time.at(0)) }
+    let (:mock_array_object) do
+      MockArrayJSONConvertible.new(mock_json_array_attribute: [
+                                     MockJSONConvertible.new(one_attribute: "Hello", other_attribute: Time.at(0)),
+                                     MockJSONConvertible.new(one_attribute: "World", other_attribute: Time.at(10))
```

Comment:
I'm thinking we might want to add a test that includes something like:
```ruby
MockJSONConvertible.new(one_attribute: "World", other_attribute: Time.at(10), array_attribute:[
  MockJSONConvertible.new.(one_attribute: "Inner World", other_attribute: Time.at(100))
])
```

Reply:
OMW